src/interface/swarm_container.hpp

```diff
@@ -131,6 +124,7 @@ class SwarmContainer {
   /// Remove a variable from the container or throw exception if not
   /// found.
   /// @param label the name of the variable to be deleted
+  /// TODO(JMM): Should we support this operation?
```

Comment:
Probably not...

Reply:
It's consistent with `meshblock_data` too so it's consistent. I just don't like we do that for either thing. Not gonna modify this now but something to keep in mind.

Reply:
Yep :+1: